app/consumer/genesis.go

```diff
@@ -114,17 +292,23 @@ func NewDefaultGenesisState(cdc codec.JSONCodec) GenesisState {
 // provider version v1,v2 or v3 to a JSON format supported by this consumer version.
 func GetConsumerGenesisTransformCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "transform [genesis-file]",
-		Short: "Transform CCV consumer genesis from an older provider version not supporting current format",
+		Use:   "transform [-to version] genesis-file",
+		Short: "Transform CCV consumer genesis data exported to a specific target format v2.x, v3.3.x, v4.x",
```

Comment:
I would recommend populating this and the supported versions strings below automatically from a slice we define outside of here, so that it's impossible to change this and the strings below in an inconsistent way